### DISCUSSION TITLE
Custom "files" folder location

### DISCUSSION THREAD

**Discussion by chencl0214** [ppy/osu#22571] [Ideas] [answered]
In order to save the maps folder continuously, I hope to support customizing or moving the location of the "files" folder, so that it does not necessarily exist in the game folder, and put it in a remote location similar to NAS, and manually identify the "files" folder in the game.

**Comment by peppy** [comment, marked as answer]
This won't be happening. You can read more about why we chose to implement things as we did [here](https://github.com/ppy/osu/wiki/User-file-storage).

**Reply by Walavouchey** [reply]
with the way they've worded it i suspect they're not talking about how the files are stored, but where. so essentially, to have the installation folder on your main drive but with the `files` folder moved to another drive since that one tends to become very large

**Reply by chencl0214** [reply]
In fact, what I want to express is that files can be read from other disks, but the answer seems to be impossible. In fact, what I want most is to rely on the account to store the id of the map, and log in to another place（another PC/iOS...） to continue to get my original Owned map to download, this problem has been raised before, but it seems that it is not easy to achieve too.Losing the map is such a pain.

**Reply by peppy** [reply]
With lazer, you can keep a `client.realm` backup. Eventually this will be enough to recover all data.
